### DISCUSSION TITLE
MHZ19 sensor in version 9.4

### DISCUSSION THREAD

**Discussion by Vglith** [arendst/Tasmota#12270] [Q&A] [answered]
Hi all,

I try to connect a MHZ 19 sensor to an sonoff with tasmota on it. However, i can't find the correct configuration for it. Is the MHZ 19 still supported in version 9.4? If so, how do I configure it correctly?

**Comment by sfromis** [comment, marked as answer]
If you use tasmota-sensors.bin.gz, the MH-Z19 serial pin configs should still be available in the gpio dropdowns in configure module/template. That's what I see for such a board, using the standard precompiled binary.

**Reply by Vglith** [reply]
Oh wauw. Now I see the MHZ TX and RX option in the configuration. So I connected the sensor but I guess the reading is a bit off hook. It says it's only 13°C while I'm in shorts and t-shirt here.  Time to check how to get this thing in a proper reading.
Thanks to get me on my way!
